package.json

```diff
@@ -15,11 +15,12 @@
     "ngrok": "ngrok http 8000"
   },
   "devDependencies": {
-    "babel-eslint": "^7.2.3",
-    "eslint": "^4.0.0",
-    "eslint-config-vacuumlabs": "^1.3.3",
-    "eslint-plugin-import": "^2.3.0",
-    "eslint-plugin-react": "^7.1.0",
+    "@babel/core": "^7.16.12",
+    "@babel/eslint-parser": "^7.16.5",
+    "eslint": "^8.8.0",
+    "eslint-config-vacuumlabs": "^2.0.14",
+    "eslint-plugin-import": "^2.25.4",
+    "eslint-plugin-react": "^7.28.0",
```

Comment:
so you updated the eslint packages - I have some questions:
- do you have eslint configured in IDE?
- have you tried running `yarn eslint . --fix` whether some changes suddenly get autofixed?
- I don't know how much experience with eslint do you have... have you by any chance checked the changes in vacuumlabs config, whether we should update something on our side?

Reply:
- yes, I have eslint and "prettier eslint" plugins in vscode
- I only compared the list of problems before and after eslint upgrade, but haven't tried fixing them.
- If we want to start minding eslint errors in this repo, I suggest setting up a circle-ci job to have them checked automatically